docs/releases/v2020.1.rst

```diff
@@ -0,0 +1,194 @@
+Gluon 2020.1
+============
+
+This is the first release of Gluon in 2020, based on OpenWrt 19.07. It
+introduces the ath79 target, which will replace ar71xx in the short
+term.
+
+Added Hardware support
+----------------------
+
+ar71xx-generic
+~~~~~~~~~~~~~~
+
+-  TP-Link Archer C60 v2
```

Comment:
I was under the impression that devices marked as broken would not be mentioned in the supported device list?

Reply:
Correct, good catch.